.github/workflows/codeql-analysis.yml

```diff
@@ -13,17 +13,17 @@ name: "CodeQL"
 
```

Comment:
changes in this file seem unrelated to the ticket. a separate pr with description of what the change is about would be better